.circleci/config.yml

```
@@ -6,6 +6,43 @@ gpu: &gpu
   environment:
     FPS_THRESHOLD: 900
 commands:
+  build_osx_conda_pkg:
+    description: "Build OSX Conda-Package"
+    parameters:
+        test:
+          description: "Do not upload to CONDA_CI_NIGHTLY"
+          type: boolean
+          default: False
+        nightly:
+          description: "Whether to build in nightly mode"
+          type: boolean
+          default: False
+    steps:
+      - run:
+          name: Conda Install OSX
+          command: |
+            curl -o ~/miniconda.sh -O https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+            chmod +x ~/miniconda.sh
+            ~/miniconda.sh -b -p $HOME/miniconda
+            rm ~/miniconda.sh
+            export PATH=$HOME/miniconda/bin:$PATH
+            conda init "$(basename "${SHELL}")"
+      - run:
+          name: Conda OSX build
+          command: |
+            export PATH=$HOME/miniconda/bin:$PATH
+            source ~/.bash_profile
+            conda activate
+            conda install -y anaconda-client ccache cmake git ninja conda-build pip
+            echo $(which -a python)
+            pip install gitpython
+            <<^ parameters.test>> anaconda login --username $AIHABITAT_CONDA_CHN --password $AIHABITAT_CONDA_CHN_PWD
```

Here channel is known and nonsensitive information, while password is taken from environment variable and parameter is used only for the variable name:

```suggestion
            anaconda login --username << parameters.AIHABITAT_CONDA_CHN >>  --password $<< parameters.AIHABITAT_CONDA_CHN_PWD_VAR >>
```